com.github.sdbg.debug.ui/plugin.xml

```diff
@@ -1077,6 +1077,15 @@
             <bundle name="com.github.sdbg.debug.core"/>
       </component>
    </extension>
+   
+   <!-- GWT Plugin debug launching -->
+   <extension
```

Comment:
Yes, exactly. Since GWT-Eclipse-Plugin will not "know" what is being launched, SDBG has to tell it to GWT-Eclipse-Plugin. This can either be done by introducing another method on the IDebugLaunch interface (say - getLaunchName() ), or by introducing a text property in the extension point itself. The latter is the "Eclipse" way, because it allows you to delay the instantiation of the IDebugInterface contribution until the user has clicked the launch link in the context menu. This - in turn - means loading of the SDBG plugins will be delayed as well until that very moment which is considered a good thing (the plugin.xml and the extension points in particular are loaded and parsed by Eclipse without loading the OSGi plugins themselves).